pom.xml

```diff
@@ -9,7 +9,7 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <jdk.version>17</jdk.version>
+        <jdk.version>11</jdk.version>
```

Comment:
why change?

Reply:
Because Maven can not build the project with default(SDK openjdk-21 and jdk version 17), so i need to change SDK to 11th version and version of jdk in POM.xml to 11th respectively. Only with such settings maven successfully builds the project

Reply:
please adjust JAVA_HOME, and settings on Project Settings to work with Java 17, it may be a problem later on in the course